artifactory/cli.go

```diff
@@ -3642,8 +3649,7 @@ func offerConfig(c *cli.Context) (*config.ArtifactoryDetails, error) {
 	if err != nil {
 		return nil, err
 	}
-	encPassword := c.BoolT("enc-password")
-	configCmd := commands.NewConfigCommand().SetDefaultDetails(details).SetInteractive(true).SetEncPassword(encPassword)
+	configCmd := commands.NewConfigCommand().SetDefaultDetails(details).SetInteractive(true).SetEncPassword(true)
```

Comment:
What is the reason that ```enc-password``` is now always true? Doesn't this mean that you won't be able to configure an Artifactory instance, which is not accessible at the time of the config?

Reply:
Since the offer config is used in generic commands, the enc-password option is not exposed. I think exposing it makes no sense. The functionality so far was reading the flag, but since it is not exposed the default value was always returned.